harmony_netcdf_to_zarr/download_utilities.py

```diff
@@ -0,0 +1,87 @@
+""" A utility module for performing multiple downloads simultaneously.
+    This is largely derived from the PO.DAAC Concise service:
+
+    https://github.com/podaac/concise/blob/develop/podaac/merger/harmony/download_worker.py
+
+"""
+from copy import deepcopy
+from logging import Logger
+from multiprocessing import Manager, Process, Queue
+from os import cpu_count
+from queue import Empty as QueueEmpty
+from typing import List
+
+from harmony.util import Config, download
+
+
+def download_granules(netcdf_urls: List[str], destination_directory: str,
+                      access_token: str, harmony_config: Config,
+                      logger: Logger, process_count: int = None) -> List[str]:
+    """ A method which scales concurrent downloads to the number of available
+        CPU cores. For further explaination, see documentation on "multi-track
```

Comment:
explanation

Reply:
Gah. That typo is just embarrassing. I'll fix that up! Thanks, Ken.